test/lit.site.cfg.in

```diff
@@ -76,6 +74,12 @@ if "@CMAKE_GENERATOR@" == "Xcode":
     config.environment['PATH'] = \
       os.path.pathsep.join((xcode_bin_dir, config.environment['PATH']))
 
+if "@LLDB_ENABLE@" == "TRUE":
+    config.available_features.add('lldb')
+    for root, dirs, files in os.walk("@LLDB_BUILD_DIR@"):
+        if root.endswith("site-packages"):
+            config.substitutions.append(('%lldb-python-path', root))
```

Comment:
Ack, will take the cue from @ddunbar. I don't mind either way, but matched it with all other substitutions used in my test.